RFCs/0002-add-pdk-config.md

```diff
@@ -157,21 +185,31 @@ Ref: (Verb-Noun Reference)[https://docs.google.com/document/d/1zX0FJBAvAIK3d3L3Q
       banana
       ```
 
-    - User-defined key, initial value:
+    - User-defined key, initial, add, then force a value:
 
       ```
-      $ pdk set config module.x.example strawberry
+      $ pdk set config --type array module.x.example strawberry
       pdk (INFO): Set initial value of modle.x.example to "strawberry"
```

Comment:
This would set the initial value to `["strawberry"]` correct?

Reply:
Yes, but the examples only show the single value.  And you wanted to avoid JSON encoding for human readable output.

If `--format=json` was set I would expect to see it as JSON array.

Reply:
I was primarily concerned with people having to _write_ JSON, but we could also just tweak this to something like:

```
pdk (INFO): Set initial value of module.x.example to array with item "strawberry"
```

Reply:
Sounds good! Will update.

Reply:
Updated